### DISCUSSION TITLE
How to Donate?

### DISCUSSION THREAD

**Discussion by Sp-cy** [Lednerb/bilberry-hugo-theme#545] [General]
How can we send some money for the coffee funds of the project maintainer? I really like this theme and seeing all the recent work porting it to v4 it's time to send some "coffee". Hope some more people could join too to keep the maintainers motivated. :D  Cheers.

**Comment by igor-baiborodine** [comment]
@Sp-cy, I'm flattered and thank you, but it's unnecessary. What motivates me is that theme's users stay happy and satisfied!

**Comment by Lednerb** [comment]
@Sp-cy @igor-baiborodine 

I totally agree with @Sp-cy that we should implement a short section in the `README.md` with your flatter Link or enabled GitHub Sponsoring (I do not have a deeper look for it so far).

You are doing a great job with this theme and I'm happy that you always bring the project to a next level, as the project is very close to my heart.

A coffee for you is more than appropriate and well earned :coffee: :+1:
